app/views/users/index.json.jbuilder

```diff
@@ -2,7 +2,7 @@ json.array! @users.each do |user|
   json.id                     user.id
   json.name                   user.name
   json.username               user.username
-  json.summary                truncate(user.summary.presence || "Active DEV author", length: 100)
+  json.summary                user.summary.mb_chars.limit(100).to_s || "Active DEV author"
```

Comment:
I'm afraid this won't work if a user's summary is empty:

```ruby
user.summary
#=> nil
truncate(user.summary.presence || "Active DEV author", length: 100)
#=> "Active DEV author"
user.summary.mb_chars.limit(100).to_s || "Active DEV author"
#=> NoMethodError: undefined method `mb_chars' for nil:NilClass
```

This version preserves the original behavior:

```suggestion
  json.summary                (user.summary.presence || "Active DEV author").mb_chars.limit(100).to_s
```